views/help.jade

```diff
@@ -5,27 +5,121 @@ block content
     h1 Help
     ul
         li
+            a(href='/help/createslide') Create Slides
+        li
+            a() Adding Songs/Translations to Glorify.cc
+        li
             a(href='#one') What is CC0 and public domain?
```

Comment:
rename this to match the question itself so that people can link to it even if the order of questions changes.

`#what-is-cc0` as an example